### PR TITLE
fix: sc4s archiving

### PR DESCRIPTION
**Pull request by ikheifets-splunk**

### CODE REVIEW

File: package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
     ]:
 
         msg = tm.render(
-            key=r,
+            key=r.lower(),
```
Comment (ikheifets-splunk) [inline package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.py:31]: we have [here](https://github.com/splunk/splunk-connect-for-syslog/blob/2f21ff9028bc44ad0b3d0dc081a066994bdd9aa3/package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.jinja#L3):

```'{{ key }}' eq "${fields.sc4s_vendor}_${fields.sc4s_product}"```

And it seems that we extracting from `SC4S_DEST_PAN_PANOS_ARCHIVE` the  `PAN_PANOS` and compare with `pan_panos`

Comment (ikheifets-splunk) [reply package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.py:31]: @rjha-splunk @mstopa-splunk probably something like that would be much better:

```match('{{ key }}' value('${fields.sc4s_vendor}_${fields.sc4s_product}') flags(ignore-case))```

than this approach

**Review by mstopa-splunk — APPROVED** [review]

**Review by rjha-splunk — APPROVED** [review]

File: package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.py (hunk 1 above)
Comment (rjha-splunk) [reply package/etc/conf.d/sc4slib/app-lp-global-archive/plugin.py:31]: This is great idea.

**Pull request merged**